lib/vmdb/appliance.rb

```diff
@@ -35,7 +35,7 @@ def self.log_config(*args)
       fh.info "RAILS Environment: #{Rails.env} version #{Rails.version}"
 
       fh.info "VMDB settings:"
-      VMDBLogger.log_hashes(fh, VMDB::Config.new("vmdb").config, :filter => Vmdb::Settings::PASSWORD_FIELDS)
+      VMDBLogger.log_hashes(fh, VMDB::Config.new("vmdb").config, :filter => Vmdb::Settings::PASSWORD_FIELDS.map(&:to_s))
```

Comment:
I think it's as simple as changing https://github.com/ManageIQ/manageiq/blob/70131632e244e41727f176acefad19d669b7ccda/gems/pending/util/vmdb-logger.rb#L92 to something like `filter = options[:filter].to_a.flatten.compact.map(&:to_s) << "password"`, since `Set`, `nil`, and `Array` all implement `.to_a`